touchforms/formplayer/api.py

```diff
@@ -279,43 +280,52 @@ def post_data(data, auth=None, content_type="application/json"):
 
     if auth:
         d['hq_auth'] = auth.to_dict()
-    domain = d.get("domain")
     if 'username' in d:
         d['restoreAs'] = d['username']
     # just default to old server for now
     return perform_experiment(d, auth, content_type)
 
 
-def perform_experiment(d, auth, content_type):
-    experiment = FormplayerExperiment(name=d["action"], context={'request': d})
+def get_candidate_session_data(control_data):
+    candidate_data = copy.deepcopy(control_data)
+    candidate_data['oneQuestionPerScreen'] = True
+    if "session_id" in control_data:
+        control_session_id = control_data["session_id"]
+    elif "session-id" in control_data:
+        control_session_id = control_data["session-id"]
+    else:
+        return True, candidate_data
```

Comment:
i'm not sure if we'll ever reach this scenario, but it looks like it would currently still run the experiment by passing formplayer whatever session id touchforms used since we didn't overwrite session_id in the `candidate_data` (which seems wrong). should this return `False` instead?

Reply:
This is the case where the request doesn't have any `session_id` associated with it yet (such as `new-form`). I don't think you could reach this block if the request had a touchforms `session_id`?

Reply:
i see, i was misreading this